scripts/install/jdownloader.sh

```diff
@@ -182,11 +182,12 @@ EOF
 _systemd
 
 # TODO: Add environment variable to bypass the following block. For unattended installs.
-if ask "Do you want to add ANY MyJDownloader account information for users?\nIt is required for them to access the web UI." N; then
-    NO_MYJD="false" # If no
-else
-    NO_MYJD="true" # If yes
-fi
+if [[ -n "${BYPASS_MYJDOWNLOADER}" ]]; then
```

Comment:
Let's stick to the `MYJD_` prefix please, it will be a lot more coherent in an env file down the road